corehq/apps/hqadmin/reports.py

```diff
@@ -1287,3 +1290,46 @@ def get_data(domains):
                 row[version_index + 1] = data.doc_count
 
         return rows.values()
+
+
+class AdminPhoneNumberReport(PhoneNumberReport):
+    name = ugettext_lazy("Admin Phone Number Report")
+    slug = 'phone_number_report'
+    fields = [
+        RequiredPhoneNumberFilter,
+    ]
+
+    dispatcher = AdminReportDispatcher
+    default_report_url = '#'
+    is_admin_report = True
+
+    @property
+    def shared_pagination_GET_params(self):
+        return [
+            {
+                'name': RequiredPhoneNumberFilter.slug,
+                'value': RequiredPhoneNumberFilter.get_value(self.request, domain=None)
+            },
+        ]
+
+    @property
+    @memoized
+    def phone_number_filter(self):
+        value = RequiredPhoneNumberFilter.get_value(self.request, domain=None)
+        if isinstance(value, basestring):
+            return value.strip()
+
+        return None
+
+    def _get_rows(self, paginate=True, link_user=True):
+        owner_cache = {}
+        if self.phone_number_filter:
+            data = PhoneNumber.objects.filter(phone_number=self.phone_number_filter)
```

Comment:
i also noticed this filter mentions you can enter a partial number but we're only matching on exact number. since this is an admin report and it won't be used too frequently i think it's ok to open this up and match using `__contains`